tests/mock_tests/mux_rollback_ut.cpp

```diff
@@ -98,6 +98,7 @@ namespace mux_rollback_test
 
             tunnel_table.set(MUX_TUNNEL, { { "dscp_mode", "uniform" },
                                            { "dst_ip", "2.2.2.2" },
+                                           { "src_ip", "3.3.3.3" },
```

Comment:
im not sure if this will be that easy, since this attribute is conditional and not required on v1.12.0 but required on v1.13.0, metadata will complain about this in both cases, that's why version check is needed i think

Reply:
I'm confused, even it's not required on 1.12 it's still a valid attribute, what would make the metadata check complain?

Reply:
hold on with this change, we will be reverting breaking change in SAI repo and this change will not be needed (@prsunny @lguohan )